scripts/ci/deploy/config/databend-query-node-1.toml

```diff
@@ -110,3 +103,45 @@ data_path = "./.databend/stateless_test_data"
 # endpoint_url = "<your-endpoint>"
 # access_key_id = "<your-key-id>"
 # access_key_secret = "<your-account-key>"
+
+
+[cache]
+
+### table meta caches ###
+# Enable table meta cache. Default is true.
+# Set it to false wll disable all the table meta caches
+enable_table_meta_caches = true
+# Max number of cached table snapshot. Set it to 0 to disable it.
+table_meta_snapshot_count = 256
+# Max number of cached table segment. Set it to 0 to disable it.
+table_meta_segment_count = 10240
+# Max number of cached table statistic meta. Set it to 0 to disable it.
+table_meta_statistic_count = 256
+
+### table bloom index caches ###
+# Enable bloom index cache. Default is true
+# Set it to false will disable all the bloom index caches
+table_index_bloom_enabled = true
```

Comment:
enable_table_bloom_index_cache?

Reply:
o , this is a typo, the name should be 'enable-table-bloom-index-caches`